js/main.js

```diff
@@ -1,70 +1,130 @@
-$(document).ready(function () {
+//Initialize Auth 0
+var lock = new Auth0Lock(
+  '7oDKP0gjAWra3ZonKyZKCstmIyzp7nK1',
+  'designsbydali.auth0.com'
+);
+
+//listenter for authenticated event
+lock.on("authenticated", function(authResult) {
+  // Use the token in authResult to getProfile() and save it to localStorage
+  lock.getProfile(authResult.idToken, function(error, profile) {
+    if (error) {
+      // Handle error
+      return;
+    }
+
+    localStorage.setItem('id_token', authResult.idToken);
+    localStorage.setItem('profile', JSON.stringify(profile));
+    loadBandApp()
+  });
+});
+
+
+$(document).ready(function() {
+  checkLoggedIn()
+  $('#login-btn').on('click', function(e) {
+    e.preventDefault()
+    lock.show()
+  })
+  $('#logout-btn').on('click', function(e) {
+    e.preventDefault()
+    logout()
+  })
   loadBands()
```

Comment:
This should prob only be called once you've checked to see if a user is logged in